dev/hal/src/Tft.cpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file Tft.cpp
+ * @author noura36
+ * @brief
+ * @version 0.1
+ * @date 2024-08-21
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+#include "mcal/inc/stm32f103xx.h"
+#include "Pin.h"
+#include "Gpio.h"
+#include "Spi.h"
+#include "Systick.h"
+#include "Tft.h"
+
+using namespace stm32::dev::mcal::gpio;
+using namespace stm32::dev::mcal::pin;
+using namespace stm32::dev::mcal::spi;
+using namespace stm32::dev::mcal::systick;
+using namespace stm32::dev::hal::tft;
+
+Tft::Tft(Pin A0, Pin rst, Spi TFtSpi) : A0(A0), rst(rst), TFtSpi(TFtSpi) {
+    Gpio::Set(A0);
+    Gpio::Set(rst);
+    //  Reset pulse
```

Comment:
Implement a `Reset` function to send a reset pulse and call it here.
you may make it `public` in case the user want to reset the display in any moment of time